doc/reference/configuration/configuration_reference.rst

```diff
@@ -492,8 +492,192 @@ The ``database`` section defines database-specific configuration parameters, suc
     | Default: false
     | Environment variable: TT_DATABASE_USE_MVCC_ENGINE
 
+..  _configuration_reference_flightrec:
 
+flightrec
+---------
 
+..  admonition:: Enterprise Edition
+    :class: fact
+
+    Configuring ``flightrec`` parameters is available in the `Enterprise Edition <https://www.tarantool.io/compare/>`_ only.
+
+
+The ``flightrec`` section describes options related to the :ref:`flight recorder <enterprise-flight-recorder>` configuration.
+Note that all options are dynamic and can be changed at runtime.
+
+..  NOTE::
+
+    ``flightrec`` can be defined in any :ref:`scope <configuration_scopes>`.
+
+To enable flight recorder, set the ``flightrec.enabled`` option to ``true`.
```

Comment:
Lets remove this sentence and add a list with links to API members instead. A user can read a how to information in the `Flight recorder` topic.